sdk/monitor/monitor-opentelemetry/package.json

```diff
@@ -112,6 +112,7 @@
     "@opentelemetry/semantic-conventions": "^1.21.0",
     "tslib": "^2.2.0",
     "@microsoft/applicationinsights-web-snippet": "^1.0.1",
+    "@microsoft/applicationinsights-web": "^3.1.0",
```

Comment:
Merge issue?

Reply:
If I didn't add this dep, the ci would fail as ```../../../common/temp/node_modules/.pnpm/@microsoft+applicationinsights-web-snippet@1.1.1/node_modules/@microsoft/applicationinsights-web-snippet/types/type.d.ts(1,32): error TS2307: Cannot find module '@microsoft/applicationinsights-web' or its corresponding type declarations.```
but now the ci still fail with other error, I don't know how to fix this

Reply:
@xirzec Could someone work on the monitor related library help on it?

Reply:
@JacksonWeber ,only update the applicationinsights-web-snippet without introducing applicationinsights-web still got this :
```../../../common/temp/node_modules/.pnpm/@microsoft+applicationinsights-web-snippet@1.1.1/node_modules/@microsoft/applicationinsights-web-snippet/types/type.d.ts(1,32): error TS2307: Cannot find module '@microsoft/applicationinsights-web' or its corresponding type declarations.```

Reply:
@kazrael2119 I spoke to the web-sdk team about this, and this issue was created https://github.com/microsoft/ApplicationInsights-JS/issues/2284. If this PR needs to get out before next week for now, we should be able to take a dev dependency on the `@microsoft/applicationinsights-web` package. If not, we can wait until next week for a fix.

Reply:
@JacksonWeber , It 's good to wait for the fix

Reply:
@kazrael2119 1.1.2 of `@microsoft/applicationinsights-web-snippet` is available now.

Reply:
update the version and revert @microsoft/applicationinsights-web change

Reply:
@JacksonWeber and could you help check the current ci failure?https://dev.azure.com/azure-sdk/public/_build/results?buildId=3552068&view=logs&j=adfbadb1-7c86-55e6-3e9b-3476e7414f1f&t=9f875c76-5c34-5697-b30e-f7aa130c0454&l=100
I don't know how to fix it, thanks

Reply:
@kazrael2119 Taking a look now. Also may be resolved by https://github.com/Azure/azure-sdk-for-js/pull/28779.